packages/app/src/cli/services/dev/processes/setup-dev-processes.test.ts

```diff
@@ -239,6 +240,98 @@ describe('setup-dev-processes', () => {
     })
   })
 
+  test('process list includes app polling when envVar is enabled', async () => {
+    process.env.SHOPIFY_CLI_ENABLE_APP_LOG_POLLING = '1'
```

Comment:
We really try to avoid this, because we'd like to be able to move tests to run in parallel. Two patterns used elsewhere:
- the function that uses environment variables might have a signature like `..., env=process.env)...`, meaning you can inject your test variable via an extra parameter.
- you can wrap up the use of `process.env` in the subject function with `getEnvironmentVariables()`, and then mock the return of `getEnvironmentVariables` here.

Should be something we lint for, so I apologise for the manual intervention!

Reply:
I saw other test that was mocking `getEnvironmentVariables` and followed that pattern 👍